server/config/index.js

```diff
@@ -37,11 +37,11 @@ const config = {
   graphQLConfig: {
     tracing: true
   },
-  graphiqlConfig: {
-    endpointURL: '/graphql', // if you want GraphiQL enabled
+  playgroundConfig: {
+    endpoint: '/graphql', // if you want GraphiQL enabled
```

Comment:
This is for consumer UI to call existing server. 
It looks like with new version users have no decision over the name of graphql server endpoint. 
Am I right?

Reply:
This is actually the config that specifies where the playground should be served. The default is `/graphql` though. We found some issues when we tried other endpoints though.